ui/components/app/signature-request/signature-request-message/signature-request-message.js

```diff
@@ -79,14 +79,14 @@ export default function SignatureRequestMessage({
         className="signature-request-message__root"
         ref={setMessageRootRef}
       >
-        <Typography
-          fontWeight={FONT_WEIGHT.BOLD}
+        <Text
+          fontWeight={FontWeight.Bold}
           color={TextColor.textDefault}
           marginLeft={4}
           className="signature-request-message__title"
```

Comment:
I think we can remove the className and the styles associated styles now that we are using the `Text` component. Could you please remove the following css from `ui/components/app/signature-request/index.scss`

```  &__title {
    @include H5;

    font-weight: 500;
  }
```

```suggestion
```

Reply:
updated